packages/renderer/src/lib/preferences/PreferencesProxiesRendering.svelte

```diff
@@ -63,19 +61,29 @@ function validate(event: any) {
   <div class="flex flex-col bg-[var(--pd-invert-content-card-bg)] rounded-md p-3 space-y-2">
     <!-- if proxy is not enabled, display a toggle -->
 
-    <SlideToggle id="toggle-proxy" bind:checked={proxyState} on:checked={event => updateProxyState(event.detail)}
-      >Proxy configuration {proxyState ? 'enabled' : 'disabled'}</SlideToggle>
+    <label for="toggle-proxy" class="inline-flex relative items-center mt-1 mb-4 cursor-pointer"
+      >Proxy configuration
+      <select
+        class="p-2 outline-none text-sm bg-charcoal-600 rounded-sm text-gray-700 placeholder-gray-700"
+        id="toggle-proxy"
+        bind:value={proxyState}>
+        <option value={0}>System</option>
+        <option value={1}>Manual</option>
+        <option value={2}>Disabled</option>
```

Comment:
```suggestion
        <option value={ProxyState.PROXY_SYSTEM}>System</option>
        <option value={ProxyState.PROXY_MANUAL}>Manual</option>
        <option value={ProxyState.PROXY_DISABLED}>Disabled</option>
```
No sure the linter would be happy but could give it a try